stock_move_packaging_qty/tests/test_stock_move_packaging_qty.py

```diff
@@ -83,3 +86,142 @@ def test_product_uom_qty_change(self):
             ],
         )
         picking.button_validate()
+
+    def test_product_packaging_qty_reserved(self):
+        """Test that the product packaging quantity reserved is correctly set."""
+        self.productA = self.env["product.product"].create(
+            {"name": "Product A", "type": "product"}
+        )
+        self.packaging_productA = self.env["product.packaging"].create(
+            {"name": "Test packaging", "product_id": self.productA.id, "qty": 5.0}
+        )
+        self.pack_location.active = True
```

Comment:
issue: please don't assign to instance variables within a test, to avoid cross-test pollution

```suggestion
        productA = self.env["product.product"].create(
            {"name": "Product A", "type": "product"}
        )
        packaging_productA = self.env["product.packaging"].create(
            {"name": "Test packaging", "product_id": self.productA.id, "qty": 5.0}
        )
        self.pack_location.active = True
```